### ISSUE TITLE
最新版本(master分支)的关系抽取task运行3个epoch f1值仍为0

### ISSUE THREAD

**Issue by ciel-zhang** [bojone/bert4keras#109]
环境：tf2.1.0、keras 2.3.1

苏神，想复现下你这个代码，用的最新master分支 ，刚刚跑的

`Epoch 2/20
2705/2705 [==============================] - 798s 295ms/step - loss: 4.8939
f1: 0.00000, precision: 1.00000, recall: 0.00000: : 21639it [04:32, 79.55it/s]
f1: 0.00000, precision: 1.00000, recall: 0.00000, best f1: 0.00000

Epoch 3/20
2705/2705 [==============================] - 800s 296ms/step - loss: 4.8942
f1: 0.00000, precision: 1.00000, recall: 0.00000: : 21639it [04:25, 81.62it/s]
f1: 0.00000, precision: 1.00000, recall: 0.00000, best f1: 0.00000`

**Comment by bojone** [comment]
试试tf 1.14？

**Comment by Areserl** [comment]
同样环境 跑的 f1也是0.000000

**Comment by cslizhang** [comment]
me too.

**Comment by bojone** [comment]
> me too.麻烦大神看看是怎么回事

没有怎么回事，我跑起来好好的。

可能因为不同环境的一些随机因素导致loss降不下去吧，可以尝试的比如换优化器、改学习率、改batch_size、换随机种子等等。

**Comment by cslizhang** [comment]
dev_pre.json文件里,spo_list_pred字段总是空,是正常吗?

**Comment by ciel-zhang** [comment]
这个问题，后来 我调通了，如果你们用的tf2，在环境变量里 设置 TF_KERAS=1 ，还有task——*.py 文件里的K，使用import tensorflow.keras.backend as K  差不多就是这样，你多调几遍，代码是没问题的 @cslizhang
